resources/js/Components/ItemIdSearchTextarea.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <cdx-field 
+      :status="validationError ? validationError.type : 'default'" 
+      :messages="validationError ? validationError.message : null"
+  >
+      <div class="progress-bar-wrapper">
+          <cdx-progress-bar v-if="loading" :aria-label="$i18n('item-form-progress-bar-aria-label')" />
+      </div>
+      <cdx-text-area
+          :label="$i18n('item-form-id-input-label')"
+          :placeholder="$i18n('item-form-id-input-placeholder')"
+          :rows="8"
+          :status="validationError ? validationError.type : 'default'"
+          v-model="textareaInputValue"
+      />
+  </cdx-field>
+</template>
+
+<script setup lang="ts">
+import { ref, inject } from 'vue';
+import { Ref } from 'vue';
+import { useI18n } from 'vue-banana-i18n';
+import { useStore } from '../store';
+import { CdxTextArea, CdxField, CdxProgressBar } from "@wikimedia/codex";
+import ValidationError from '../types/ValidationError';
+
+// Run it with compat mode
+// https://v3-migration.vuejs.org/breaking-changes/v-model.html
+CdxTextArea.compatConfig = {
+    ...CdxTextArea.compatConfig,
+    COMPONENT_V_MODEL: false,
+};
+
+const validationError: Ref<ValidationError> = ref(null);
+
+const banana = useI18n();
+
+const store = useStore();
+const textareaInputValue = ref(store.lastSearchedIds);
+
+const MAX_NUM_IDS = inject('MAX_NUM_IDS');
+
+defineProps<{loading: boolean}>();
+
+function splitInput(): Array<string> {
+    return textareaInputValue.value.split( '\n' );
+}
```

Comment:
I think some spaces are needed in this line and after the next couple of functions. So close together is a bit messy on the eyes.

Reply:
I've added some spaces.